Tasks/AzureAppServiceManage/azureappservicemanage.ts

```diff
@@ -153,6 +153,7 @@ async function run() {
     }
 
     if (!taskResult) {
+        console.log("##vso[task.logissue type=error;code=AASM_Task_InternalError;]" + errorMessage);
```

Comment:
Don't log entire errorMessage. This can have GDPR data. The comment is applicable to all instances where this is being done in this PR.